packages/autocomplete-core/src/utils/isChrome.ts

```diff
@@ -1,14 +1,12 @@
 const chromeRegex = /(crmo|crios|chrome)/i;
 const edgeRegex = /edg(e|ios|a)?/i;
-const samsungBrowserRegex = /samsungbrowser/i;
 const chromiumRegex = /chromium/i;
 
 export function isChrome(userAgent: string) {
   return Boolean(
     userAgent &&
       userAgent.match(chromeRegex) &&
       !userAgent.match(edgeRegex) &&
-      !userAgent.match(samsungBrowserRegex) &&
```

Comment:
actually I think the util is correct, but we shouldn't use isChrome, but only isSamsung, what do you think?

Reply:
I'm saying that, as Samsung browser isn't the same as chrome

Reply:
Indeed this seems more appropriate, should we delete isChrome util then?
And do the check on all Samsung devices as you mentioned

Reply:
yes, that seems more correct to me